Swiften/Parser/PayloadParsers/UnitTest/MIXParticipantParserTest.cpp

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2017 Tarun Gupta
+ * Licensed under the simplified BSD license.
+ * See Documentation/Licenses/BSD-simplified.txt for more information.
+ */
+
+#include <gtest/gtest.h>
+
+#include <Swiften/Elements/MIXParticipant.h>
+#include <Swiften/Parser/PayloadParsers/UnitTest/PayloadsParserTester.h>
+
+using namespace Swift;
+
+TEST(MIXParticipantParserTest, XEP0369_Example1_ParticipantWithNick) {
+    PayloadsParserTester parser;
+    ASSERT_TRUE(parser.parse(
+                "<participant xmlns='urn:xmpp:mix:0'> <nick>thirdwitch</nick> </participant>"
+                ));
+
+    MIXParticipant::ref payload = parser.getPayload<MIXParticipant>();
+    ASSERT_TRUE(payload != NULL);
```

Comment:
Never use NULL in C++11 (or later), always nullptr.
(In this case, I think you can simply omit the comparison and assert on (payload).